content/docs/SUSHI/running/_index.md

```diff
@@ -15,27 +15,58 @@ This documentation assumes you have a SUSHI-compliant project structure and conf
 SUSHI is executed from the command line. The general form of the SUSHI execution command is as follows:
 
 ```shell
-{{< terminal >}} sushi {specification-directory} {options}
+{{< terminal >}} sushi {command} {options}
+```
+
+Use `sushi --version` and `sushi help` to get basic information about the current SUSHI version.
+
```

Comment:
I think it might be nice to have an alert box here that briefly notes that SUSHI 2.x did not use sub-commands and that, where possible, SUSHI 3.x tries to honor the SUSHI 2.x approach.  E.g.

> NOTE: Previous versions of SUSHI did not use sub-commands such as `build` and `init`. While SUSHI users are encouraged to use the new sub-commands, SUSHI will attempt to properly handle legacy usage when a sub-command is not provided.

Reply:
Hmm this is the PR that is going to be the most difficult to indicate differences between 3.x and earlier versions. I was considering ways to toggle back to the earlier versions within the documentation, but I think this is probably good to have in the 3.x and later documentation regardless, so I can add it here.

Reply:
Yeah, as I was reviewing this PR, I was also thinking that this is the one that will present the most challenges for supporting 2.x and 3.x at once. I think maybe we punt on doing anything too fancy and just have two sections near the top: `Running SUSHI 2.x` and `Running SUSHI 3.0 Beta`.

Then once we officially release 3.0 Beta, we have mostly 3.0 info, but some sections have a note. E.g., _In SUSHI 2.x, use `sushi --init` to initialize a new project._

Reply:
As part of the other task I had also been playing around with options like expanding/collapsing the 2.x documentation or having a tab for the old version.

Regardless, I think it still makes sense to add that alert into this PR. Assuming you still agree, I'll add it to this PR.

Reply:
Added in [95b59be](https://github.com/FSHSchool/site/pull/72/commits/95b59be945240e7c93c97be94cc9e649062d7de0).